### ISSUE TITLE
[NotImplementedError] Can't load newest version world (beta, also tried on some older worlds non beta)

### ISSUE THREAD

**Issue by Hanprogramer** [BluCodeGH/bedrock#7]
I am just trying to open the world. 
OS: Windows 10
Minecraft version 1.16.230 beta (also tried on older world)
Running on Python 3.8

Edit:: After testing for a way older version i found that it works. But i do need for the latest version
Edit2:: found that chunk version 15 works, then it is probably cuz of the beta
```
    def load_world(self,path):
        with bedrock.World(path) as world:
            print(world.getBlock(0,0,0))
            # block = bedrock.Block("minecraft:stone") # Name must include `minecraft:`
            # world.setBlock(0, 0, 0, block)
            # block = bedrock.Block("minecraft:stone", 2) # Data value
            # world.setBlock(0, 1, 0, block)
            # if world.getBlock(0, 2, 0).name == "minecraft:stone":
            #     print("More stone!")
            # Autosave on close.
```
The error message:
```
File "main.py", line 829, in on_choose_world
    self.load_world(path)
  File "main.py", line 882, in load_world
    print(world.getBlock(0,0,0))
  File "C:\Program Files (x86)\Python37-32\lib\site-packages\bedrock-0.1-py3.7.egg\bedrock\bedrock.py", line 39, in getBlock        
    chunk = self.getChunk(cx, cz)
  File "C:\Program Files (x86)\Python37-32\lib\site-packages\bedrock-0.1-py3.7.egg\bedrock\bedrock.py", line 30, in getChunk        
    chunk = Chunk(self.db, x, z)
  File "C:\Program Files (x86)\Python37-32\lib\site-packages\bedrock-0.1-py3.7.egg\bedrock\bedrock.py", line 78, in __init__        
    self.version = self._loadVersion(db)
  File "C:\Program Files (x86)\Python37-32\lib\site-packages\bedrock-0.1-py3.7.egg\bedrock\bedrock.py", line 101, in _loadVersion   
    raise NotImplementedError("Unexpected chunk version {} at chunk {} {}.".format(version, self.x, self.z))
NotImplementedError: Unexpected chunk version 22 at chunk 0 0.
```

**Comment by BluCodeGH** [comment]
Bedrock will need a nontrivial rewrite to support the latest beta due to the world height increase. From what I have seen the subchunk format is still backwards compatible, but I will need to find some time to investigate further and make the fixes.

**Comment by BluCodeGH** [comment]
This should be resolved now that 1.17 support has been added!
